cmd/src/code_intel_upload_flags.go

```diff
@@ -122,56 +109,52 @@ func parseAndValidateCodeIntelUploadFlags(args []string) (*output.Output, bool,
 	// and maybe we'll use some in the future
 	codeintelUploadFlags.apiFlags = api.NewFlags(apiClientFlagSet)
 	if err := apiClientFlagSet.Parse(insecureSkipVerifyFlag); err != nil {
-		return nil, false, err
+		return nil, err
 	}
 
 	if !isFlagSet(codeintelUploadFlagSet, "file") {
 		defaultFile, err := inferDefaultFile()
 		if err != nil {
-			return nil, false, err
+			return nil, err
 		}
 		codeintelUploadFlags.file = defaultFile
 	}
 
 	// Check to see if input file exists
 	if _, err := os.Stat(codeintelUploadFlags.file); os.IsNotExist(err) {
 		if !isFlagSet(codeintelUploadFlagSet, "file") {
-			return nil, false, formatInferenceError(argumentInferenceError{"file", err})
+			return nil, formatInferenceError(argumentInferenceError{"file", err})
 		}
 
-		return nil, false, errors.Newf("file %q does not exist", codeintelUploadFlags.file)
+		return nil, errors.Newf("file %q does not exist", codeintelUploadFlags.file)
 	}
 
 	isSCIPAvailable, err := isSCIPAvailable()
 	if err != nil {
-		return nil, false, err
+		return nil, err
```

Comment:
Let's remove the `isSCIPAvailable()` check here? I don't think we particularly care about what error message someone gets on having a large divergence between src-cli and Sourcegraph.

We also have clear docs on how to get a compatible src-cli version. https://github.com/sourcegraph/src-cli?tab=readme-ov-file#version-compatible-with-your-sourcegraph-instance

The logic for the `isSCIPAvailable` function is a bit out-dated with this PR anyways (it still references `lsif`).